Explorer/Assets/Scripts/ECS/StreamableLoading/Systems/ConcludeTextureLoadingSystem.cs

```diff
@@ -12,7 +12,12 @@ public partial class ConcludeTextureLoadingSystem : ConcludeLoadingSystemBase<Te
     {
         internal ConcludeTextureLoadingSystem(World world) : base(world) { }
 
-        protected override Texture2D GetAsset(UnityWebRequest webRequest) =>
-            DownloadHandlerTexture.GetContent(webRequest);
+        protected override Texture2D GetAsset(UnityWebRequest webRequest, in GetTextureIntention getTextureIntention)
+        {
+            Texture2D tex = DownloadHandlerTexture.GetContent(webRequest);
```

Comment:
Will we be investigating moving this to a zero allocation solution writing to existing texture buffers?

Reply:
This implementation stands for GPU-incompatible formats, for them, we can't call `LoadRawTextureData`. I think we need to switch to converted formats in order to benefit from preallocated buffers.
When it comes to `DownloadHandlerTexture`

> DownloadHandlerTexture stores received data in a pre-allocated Unity [Texture](https://docs.unity3d.com/ScriptReference/Texture.html) object. It is optimized for downloading images from Web servers, and performs image decompression and decoding on a worker thread.

unless `decompression and decoding on a worker thread` API is exposed, this is the best way of doing things; I will ask Unity